src/export/option.ts

```diff
@@ -108,8 +108,11 @@ interface ToolboxComponentOption extends ToolboxOption {
     }
 }
 
+export { SliderDataZoomOption, InsideDataZoomOption };
```

Comment:
Use `as` to rename to `SliderDataZoomComponentOption`, `InsideDataZoomComponentOption` to keep a consistent convention.

Reply:
✔️